.github/pull_request_template.md

```diff
@@ -1,39 +1,46 @@
 ## Description
 
-A detailed explanation of the changes in your PR.
+<!-- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
 
-## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
+All three items *must be* checked:
+<!-- Remove conflicting items that won't be checked. -->
```

Comment:
This now seems like it's part of Description and so I'm starting to mentally prepare myself PRs that look like this:

![image](https://github.com/stackrox/stackrox/assets/537715/85aae09a-c89c-47fb-b717-d8826410002e)

Not great, in my opinion.
I'd suggest re-introducing a section for these items.

I better thing would be to move "CI results" away from here and call this section for documentation. 
```suggestion
## User-facing documentation

Here must be *two* items and both *must be* checked:
<!-- Remove conflicting items that won't be checked. -->
```